var/spack/repos/builtin/packages/dune-common/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class DuneCommon(PythonPackage):
+    """Dune, the distributed and unified numerics environment."""
+
+    homepage = "https://www.dune-project.org/doc/gettingstarted/"
+    git = "https://gitlab.dune-project.org/core/dune-common"
+    pypi = "dune-common/dune-common-2.9.0.tar.gz"
+
+    version("2.9.0", sha256="35e6ede3da10be5c665db4cabb2bf4bd76a2e483255b2fca63c0a9d5ecad064c")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("cmake@3.13.0:", type="build")
+    depends_on("mpi@2:")
+    depends_on("pkgconf", type="build")
+    depends_on("gcc@7:", type="build")
```

Comment:
Missing many dependencies. Build time:
```toml
requires = ['cmake>=3.13', 'jinja2', 'ninja', 'numpy', 'pip>=21', 'portalocker', 'requests', 'scikit-build', 'setuptools>=41', 'wheel']
```
Run-time:
```
jinja2
numpy
pip>=21
portalocker
setuptools>=41
wheel
```

Reply:
Hi. How can i add this dependencies to the package? Thanks.

Reply:
The docs explain the differences and show how to specify a dependency with multiple types when appropriate.  See https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-types.

Most python package names start with `py-` but the documentation also discusses the decision at https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name.